app/javascript/utilities/documentation/index.js

```diff
@@ -0,0 +1,160 @@
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+const sass = require('node-sass');
+const CSSOM = require('cssom');
+const prettier = require('prettier');
+const renderCss = util.promisify(sass.render);
+const file = fs.promises;
+
+const GENERATED_STORIES_FOLDER = path.join(
+  __dirname,
+  '../../generated_stories/__stories__',
+);
+
+/**
+ * Generates a style sheet object for the given SASS/CSS file.
+ *
+ * @param {string} file The file to load as a style sheet.
+ *
+ * @returns {CSSStyleSheet} The stylesheet for the given file.
+ */
+async function getStyleSheet(file) {
+  const { css: bytes } = await renderCss({
+    file,
+  });
+  const utilityClassesContent = new TextDecoder('utf-8').decode(bytes);
+  const styleSheet = CSSOM.parse(utilityClassesContent);
```

Comment:
I got the idea to use cssom after looking up parsing a CSS abstract syntax tree ([AST](https://en.wikipedia.org/wiki/Abstract_syntax_tree)) on [astexplorer.net](https://astexplorer.net)